pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
```

Comment:
Have you tested this with `mvn hpi:run` and actual Jenkins? I am afraid this will break some of that.